dask/dask.yaml

```diff
@@ -1,5 +1,7 @@
 temporary-directory: null     # Directory for local disk like /tmp, /scratch, or /local
 
+shuffle-disk-compression: null  # compression for on disk-shuffling. Partd supports ZLib, BZ2, SNAPPY, BLOSC
```

Comment:
This should probably be under a "dataframe" key.

Reply:
Already implemented